lessons/codap-exploring/langs/en-us/index.adoc

```diff
@@ -122,7 +116,8 @@ Brainstorm other scenarios where Booleans are useful in and out of the programmi
 === Investigate
 [.lesson-instruction]
 --
-In pairs, complete @printable-exercise{pages/booleans.adoc}, making predictions about what a variety of Boolean expressions will return and testing them in the editor.
+- Turn to @printable-exercise{pages/booleans.adoc}.
```

Comment:
Not sure why, but this page is not listed in the pull request. I made a few formatting changes to make directions clearer and bring it in line with updates made to the original page from the simple datatypes lesson.